### ISSUE TITLE
SC3.3.8: Redundant entry - are redundant resumes/curriculum vitae in online job applications a failure?

### ISSUE THREAD

**Issue by NoahGresser** [w3c/wcag#1293]
Often times online job applications require you to attach your resume/curriculum vitae as a document, and then re-enter a large amount of information that would already be in the resume/curriculum vitae into individual input fields. 

Would this be considered a failure of the redundant entry criterion? 
Or should it be added as one of the "broader exceptions" asked about in the editor's note? 
> Editors' note: Are there broader exceptions needed than essential? E.g. for mandated or required information re-entry.

Either way, it seems like the practice should at least be brought up in the understanding page for the criterion, given that it sounds as though it should be covered by this criterion, and is so prevalent with online job applications.

**Comment by wardav** [comment]
My personal opinion with this is that the SC should probably be in scope when the person is entering the same information **and** is entering it in the same way, or more broadly in the same format. 

For example, if someone is asked to upload a picture of some photo ID (e.g. for AirBNB), to prove their identity, which also has their date of birth one. Is it a requirement that the website OCR the picture and autofill any matching fields?

I feel like that's a stretch and so there should be a broad exception that it is not in scope when there would **have** to be some sort of conversion or processing. For example, extracting the text from a PDF, getting something from a picture.

**Comment by alastc** [comment]
Hi Folks, we agreed an update to the understanding document to make that clearer, hopefully the linked PR makes sense for you?
